homeassistant/components/min_max/config_flow.py

```diff
@@ -72,3 +75,45 @@ class ConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
     def async_config_entry_title(self, options: Mapping[str, Any]) -> str:
         """Return config entry title."""
         return cast(str, options["name"]) if "name" in options else ""
+
+    @callback
+    @staticmethod
+    def async_setup_preview() -> None:
+        """Set up preview WS API."""
+        hass = async_get_hass()
+        websocket_api.async_register_command(hass, ws_preview_min_max)
+
+
+@callback
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "min_max/preview",
+        vol.Required("flow_id"): str,
+        vol.Required("flow_type"): str,
+        vol.Required("user_input"): dict,
```

Comment:
Should we make this `CONFIG_SCHEMA` ? 
```suggestion
        vol.Required("user_input"): CONFIG_SCHEMA,
```

Reply:
oh I guess it depends on the flow_type.